var/spack/repos/builtin/packages/r-genomeinfodbdata/package.py

```diff
@@ -13,8 +13,12 @@ class RGenomeinfodbdata(RPackage):
     homepage = "https://bioconductor.org/packages/GenomeInfoDbData/"
     url      = "https://bioconductor.org/packages/3.5/data/annotation/src/contrib/GenomeInfoDbData_0.99.0.tar.gz"
 
+    version('1.2.1', sha256='75e6d683a29b8baeec66ba5194aa59a6aa69b04fae5a9c718a105c155fb41711',
+            url='https://bioconductor.org/packages/3.9/data/annotation/src/contrib/GenomeInfoDbData_1.2.1.tar.gz')
     version('1.1.0', sha256='6efdca22839c90d455843bdab7c0ecb5d48e3b6c2f7b4882d3210a6bbad4304c',
             url='https://bioconductor.org/packages/3.7/data/annotation/src/contrib/GenomeInfoDbData_1.1.0.tar.gz')
     version('0.99.0', sha256='457049804bbd70f218c1c84067a23e83bdecb7304a3e4d8b697fee0b16dc1888')
+
     depends_on('r@3.4.0:3.4.9', when='@0.99.0', type=('build', 'run'))
     depends_on('r@3.5.0:3.5.9', when='@1.1.0', type=('build', 'run'))
+    depends_on('r@3.6.0:3.6.9', when='@1.2.1', type=('build', 'run'))
```

Comment:
It looks like this package wasn't updated to use the new git checkout and R versions

Reply:
GenomeInfoDbData doesn't have a remote git for some reason. It is only accessible via the tarball -- I'll update it to follow the same dependency pattern as the others.

Reply:
That's so weird...

Reply:
Ping, still needs to be updated I think.

Reply:
Fixed!